src/EditorFeatures/CSharpTest/QualifyMemberAccess/QualifyMemberAccessTests.cs

```diff
@@ -641,6 +681,54 @@ void M()
 CodeStyleOptions.QualifyMethodAccess);
         }
 
+        [WorkItem(28509, "https://github.com/dotnet/roslyn/issues/28509")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsQualifyMemberAccess)]
+        public async Task QualifyMethodAccess_NotSuggestedOnObjectInitializer()
+        {
+            await TestMissingAsyncWithOption(
+@"class C
+{
+    void M()
+    {
+         var foo = 1;
+         var test = new System.Collections.Generic.List<int> { [|foo|] };
+    }
+}",
+CodeStyleOptions.QualifyMethodAccess);
+        }
+
+        [WorkItem(28509, "https://github.com/dotnet/roslyn/issues/28509")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsQualifyMemberAccess)]
+        public async Task QualifyLocalMethodAccess_NotSuggestedOnObjectInitializer()
```

Comment:
❕ This is a collection initializer, not an object initializer (test is correct, but the method name here is wrong)